src/dlgTriggerEditor.cpp

```diff
@@ -618,10 +618,10 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
 
     treeWidget_searchResults->setColumnCount(4);
     QStringList labelList;
-    labelList << tr("Type", "Heading for the first column of the search results display treewidget")
-              << tr("Name", "Heading for the second column of the search results display treewidget")
-              << tr("Where", "Heading for the third column of the search results display treewidget")
-              << tr("What", "Heading for the four column of the search results display treewidget");
+    labelList << tr("Type", "Heading for the first column of the search results")
+              << tr("Name", "Heading for the second column of the search results")
+              << tr("Where", "Heading for the third column of the search results")
+              << tr("What", "Heading for the forth column of the search results");
```

Comment:
`fourth`

Reply:
:woman_facepalming: Yeah - forth is something else entirely...

Reply:
Otherwise - is good, yes?

Reply:
Yup